lib/siwapp_web/router.ex

```diff
@@ -93,6 +93,12 @@ defmodule SiwappWeb.Router do
     get "/users/settings", UserSettingsController, :edit
     put "/users/settings", UserSettingsController, :update
     get "/users/settings/confirm_email/:token", UserSettingsController, :confirm_email
+
+    get "/series", SeriesController, :index
+    get "/series/edit/:id", SeriesController, :edit
+    put "/series/edit/:id", SeriesController, :update
+    get "/series/new", SeriesController, :new
+    post "/series/new", SeriesController, :create
```

Comment:
Instead of specify each of these routes you can use `resources` and exclude the ones you're not interested. 
See https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4